### ISSUE TITLE
Ch 2 : Training set, Test set and Data preprocessing 

### ISSUE THREAD

**Issue by heeheehoho** [heeheehoho/My-AI-Study#3]
- Supervised Learning, Unsupervised Learning
- Training Set, Test Set
- Evaluation of Training Model

**Comment by heeheehoho** [comment]
.
